Example/ViewController.swift

```diff
@@ -49,6 +49,12 @@ extension ViewController: CustomDropDownDelegate, CustomDropDownDataSource {
     func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath, data: Any, identifier: Int) {
         
     }
+    
+    func config(identifier: Int) -> DropDownConfig {
```

Comment:
As the default value is true for dropDownCollapsable, we dont need this block of code. Remove this